verify/swrs/transform/materialized_view/report_001@v1.3.0.sql

```diff
@@ -0,0 +1,10 @@
+-- Verify ggircs:swrs/transform/materialized_view/report_001 on pg
+
+begin;
+
+-- selecting from the matview will throw an error if it doesn't exist
+-- but there's no need to return any value so select where false
+select * from swrs_transform.report where false;
```

Comment:
I think the sql way of writing this is `select 1 from swrs_transform.report where false;` so it doesn't even have to scan the table columns (but it doesn't matter, especially for verify)